packages/message-box/src/message-box.d.ts

```diff
@@ -0,0 +1,177 @@
+import Vue, { VNode } from 'vue'
+export type MessageType = 'success' | 'warning' | 'info' | 'error'
+export type MessageBoxCloseAction = 'confirm' | 'cancel' | 'close'
+export type MessageBoxData = MessageBoxInputData
```

Comment:
I think we can use a `d.ts` file to export all types

Reply:
this is message-box.d.ts